src/D2L.Security.OAuth2/Validation/Request/IRequestAuthenticator.cs

```diff
@@ -23,7 +23,7 @@ public interface IRequestAuthenticator {
 			HttpRequestMessage request
 		);
 
-#if !DNXCORE50
+#if !NET5_0

```

Comment:
`System.Web.HttpRequest` is only a thing in .NET Framework.
The overload above does `HttpRequestMessage` which is the cross-platform thing.